packages/client/src/views/Organization/ListOfOrganizations.tsx

```diff
@@ -0,0 +1,245 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import React from 'react'
+import { Frame } from '@opencrvs/components/lib/Frame'
+import { Header } from '@client/components/Header/Header'
+import { navigationMessages } from '@client/i18n/messages/views/navigation'
+import { constantsMessages } from '@client/i18n/messages'
+import { Navigation } from '@client/components/interface/Navigation'
+import { useIntl } from 'react-intl'
+import { Pagination } from '@opencrvs/components/lib/Pagination'
+import {
+  Content,
+  Link,
+  ListViewItemSimplified,
+  ListViewSimplified,
+  NoResultText,
+  Stack
+} from '@opencrvs/components'
+import { useDispatch, useSelector } from 'react-redux'
+import { IStoreState } from '@client/store'
+import { ILocation, LocationType } from '@client/offline/reducer'
+import { useParams } from 'react-router'
+import {
+  goToOrganizationList,
+  goToPerformanceHome,
+  goToTeamUserList
+} from '@client/navigation'
+import { Activity, User } from '@opencrvs/components/lib/icons'
+import { Button } from '@opencrvs/components/lib/Button'
+import startOfMonth from 'date-fns/startOfMonth'
+import subMonths from 'date-fns/subMonths'
+
+const DEFAULT_PAGINATION_LIST_SIZE = 10
+
+type IRouteProps = {
+  index: string
+}
+
+type IGetNewLevel = {
+  currentLevel: null | ILocation
+  dataLevel: ILocation[]
+}
+
+export function ListOfOrganizations() {
+  const intl = useIntl()
+  const { index } = useParams<IRouteProps>()
+  const dispatch = useDispatch()
+  //
+  const getNewLevel =
+    (index: string) =>
+    (store: IStoreState): IGetNewLevel => {
+      const indexString = index ?? '0'
+      const locations = store.offline.offlineData.locations as {
+        [key: string]: ILocation
+      }
+      const offices = store.offline.offlineData.offices as {
+        [key: string]: ILocation
+      }
+
+      let dataLevel = Object.values(locations).filter(
+        (s) => s.partOf === `Location/${indexString}`
+      )
+
+      let currentLevel = null
```

Comment:
This would be better named as `location` or `currentlySelectedLocation` or similar for better readability